pkg/operator/configobservation/auth/auth_serviceaccountissuer.go

```diff
@@ -116,6 +129,11 @@ func unstructuredConfigForIssuer(issuer string) map[string]interface{} {
 			"service-account-issuer": []interface{}{
 				issuer,
 			},
+			"service-account-jwks-uri": []interface{}{
+				// We need to set the URL so that it points to our LB and doesn't
+				// default to KAS IP which isn't included in the serving certificate.
+				internalLBURL + "/openid/v1/jwks",
```

Comment:
If sts or cco were to override the default issuer, would they need to be able to set this value or at least have it be cleared?

Also, maybe an update to the [enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/kube-apiserver/bound-sa-tokens.md) is suggested?

Reply:
You're right. If I understand the logic of this observer, this would have to be moved to the condition above when no issuer is set, correct?